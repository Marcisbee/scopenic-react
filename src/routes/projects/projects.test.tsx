import { shallow } from 'enzyme';
import React from 'react';
import Projects from './projects';

describe('Projects', () => {
  it('renders without crashing', () => {
    const component = shallow(<Projects />);
    expect(component).toMatchSnapshot();
  });
});
