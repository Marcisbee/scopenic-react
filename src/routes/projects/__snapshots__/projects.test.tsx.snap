// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders without crashing 1`] = `
<div>
  <Header />
  <h1>
    Projects
  </h1>
</div>
`;
